### PR TITLE
Added cypress test for links and markdownShortcuts

### PR DESCRIPTION
**Pull request by anishagg17**
**Description**
 Added cypress test for `links` and `markdown shortcuts`

**Checks**
- [ ] The new code matches the existing patterns and styles.
- [ ] The tests pass with `yarn test`.
- [ ] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [ ] The relevant examples still work. (Run examples with `yarn start`.)
- [ ] You've [added a changeset](https://github.com/atlassian/changesets/blob/master/docs/adding-a-changeset.md) if changing functionality. (Add one with `yarn changeset add`.)

### CODE REVIEW

File: cypress/integration/links.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+describe('On links example', () => {
```
Comment (timbuckley) [inline cypress/integration/links.ts:1]: The `describe` is the noun and the `it` is the verb. So like `describe('Monkey', () => { it('can climb')})`
```suggestion
describe('Links example', () => {
```

File: cypress/integration/markdownShortcuts.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    cy.visit('examples/markdown-shortcuts')
+  })
+
+  it('contains quote', () => {
+    cy.findByRole('textbox')
+      .find('blockquote')
+      .should('contain.text', 'A wise quote.')
+  })
+
+  it('add list items', () => {
```
Comment (timbuckley) [inline cypress/integration/markdownShortcuts.ts:12]:
```suggestion
  it('can add list items', () => {
```

File: cypress/integration/markdownShortcuts.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      .should('contain.text', '1st Item')
+      .get('ul > li')
+      .eq(1)
+      .should('contain.text', '2nd Item')
+      .get('ul > li')
+      .eq(2)
+      .should('contain.text', '3rd Item')
+  })
+
+  it('add a h1 item', () => {
```
Comment (timbuckley) [inline cypress/integration/markdownShortcuts.ts:34]:
```suggestion
  it('can add a h1 item', () => {
```

File: cypress/integration/readonly.ts [hunk 4]
```diff
@@ -1,7 +1,10 @@
 describe('readonly editor', () => {
-  it('should not be editable', () => {
+  beforeEach(() => {
     cy.visit('examples/read-only')
-      .get('[data-slate-editor="true"]')
+  })
+
+  it('should not be editable', () => {
+    cy.get('[data-slate-editor="true"]')
```
Comment (timbuckley) [inline cypress/integration/readonly.ts:7]: Put this string in a variable like `slateEditor`

**Review by timbuckley — APPROVED** [review]
Looks good after you address my comments. Feel free to merge!

File: cypress/integration/links.ts [hunk 5]
```diff
@@ -0,0 +1,11 @@
+describe('Links example', () => {
```
[the remark this answers is not in the document]
Comment (anishagg17) [reply cypress/integration/links.ts:1]: Moved to `cypress/integration/examples/` 👍

**Pull request merged**